Makefile

```diff
@@ -54,8 +74,10 @@ dev_manifest:
 
 .PHONY: generate
 generate:
-	go generate ./pkg/... ./cmd/... && \
-	openapi-gen --input-dirs github.com/kubeflow/mpi-operator/pkg/apis/kubeflow/v2beta1,k8s.io/api/core/v1,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/api/resource,k8s.io/apimachinery/pkg/runtime,k8s.io/apimachinery/pkg/util/intstr,k8s.io/apimachinery/pkg/version,github.com/kubeflow/common/pkg/apis/common/v1 --output-package github.com/kubeflow/mpi-operator/pkg/apis/kubeflow/v2beta1 --go-header-file hack/boilerplate/boilerplate.go.txt
+	go generate ./pkg/... ./cmd/...
```

Comment:
why is this changing?

Reply:
L58 duplicates the below:

https://github.com/kubeflow/mpi-operator/blob/cd83424f65dd6b2a57c32a2fde95a6edd649943e/hack/python-sdk/gen-sdk.sh#L34-L35

So I removed L58.